cmd/thermal-recorder/main.go

```diff
@@ -190,7 +191,7 @@ func logConfig(conf *Config) {
 	log.Printf("location latitude: %v", conf.Location.Latitude)
 	log.Printf("location longitude: %v", conf.Location.Longitude)
 	if conf.Recorder.UseSunriseSunsetWindow {
-		log.Printf("recording window using sunrise sunset with delay of %v minutes", conf.Recorder.SunriseSunsetDelay)
+		log.Printf("recording window using sunrise sunset with a sunrise offset of %v minutes a sunet offset of %v minutes", conf.Recorder.SunriseOffset, conf.Recorder.SunsetOffset)
```

Comment:
`recording window is based on sunrise and sunset (sunrise offset is %v minutes, sunset offset is %v minutes)`